build-tools/api-xml-adjuster/README.md

```diff
@@ -0,0 +1,32 @@
+# Mono.Android API XML generator
```

Comment:
I don't think so. The contents of this README.md is really specific to *this* exact directory and does not make sense in any other physical directory.